src/styles/Modal.css

```diff
@@ -37,3 +37,41 @@
 .cancelDelete:hover {
   background-color: #ccc;
 }
+
+.detailsModal {
+  margin-bottom: 20px;
+  padding: 30px 20px;
+  background-color: #98c1d9;
+  border: 1px solid #293241;
+  border-radius: 6px;
+}
+.detailsModal h1 {
+  margin-top: 0;
+}
+.detailsModal h3 {
+  /* display: inline-block;
+  text-align: left; */
+  margin: 0;
+}
+.detailsModal span {
+  /* margin-left: 10px; */
+  margin-top: 4px;
+  padding: 10px;
+}
+.detailsField {
+  display: flex;
+  align-items: center;
+  padding-left: 3px;
+  padding-right: 3px;
+}
```

Comment:
The styling of the details is nice. Gives the user quick straightforward information on what they need to know.